.github/workflows/unitests.yml

```diff
@@ -12,20 +12,20 @@ jobs:
     steps:
       - name: Install packages
         run: |
-          sudo apt update
+          sudo apt-get update
```

Comment:
seems unnecessary?

Reply:
This is a minor fix. Using the apt command in scripts is discouraged, and it is recommended to use apt-get instead. According to the apt manual:

> The apt(8) commandline is designed as an end-user tool and its behavior might change between versions. Although it aims to maintain backward compatibility, this is not guaranteed if a modification appears advantageous for interactive use.

That's why a warning message is issued when apt is employed in a script.

However, this fix can be safely ignored; nevertheless, there isn't a reason to ignore it.

Reply:
it has pipeline here, maybe it's ok.